config/systemjs.config.js

```diff
@@ -32,12 +32,13 @@ System.config({
   },
 
   map: {
-    "angular2": "npm:angular2@2.0.0-beta.11",
+    "angular2": "npm:angular2@2.0.0-beta.12",
```

Comment:
So if you upgrade this to `beta.12`, and the package.json version is still at `beta.9`... which one does it pick?